workers/workers_test.go

```diff
@@ -63,9 +97,8 @@ func TestWorkerStartAndStop(t *testing.T) {
 			t.FailNow()
 		}
 
-		time.Sleep(minTickForTest * 100)
-		if !assert.True(t, start.Load() > 0, "Cannot start worker") {
-			t.FailNow()
+		if !tryTestGtZero(start) {
+			assert.FailNow(t, "Cannot start worker")
```

Comment:
there is 6 `"Cannot start worker"` errors , maybe move them to var?

Reply:
mb "Cannot start worker" or "Cannot start workers"